src/zcash/Note.cpp

```diff
@@ -188,6 +188,23 @@ std::optional<SaplingNote> SaplingNotePlaintext::note(const SaplingIncomingViewi
     }
 }
 
+std::pair<SaplingNotePlaintext, SaplingPaymentAddress> SaplingNotePlaintext::from_rust(
+    rust::Box<wallet::DecryptedSaplingOutput> decrypted)
+{
+    SaplingPaymentAddress pa(
+        decrypted->recipient_d(),
+        uint256::FromRawBytes(decrypted->recipient_pk_d()));
```

Comment:
I checked that this is effectively little-endian as required (similarly for `rseed`).